<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QWidget" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gamewindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLCDNumber" name="linesLCD">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>128</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="linesLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Muna PUA</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 71, 71);</string>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </widget>
  <widget class="QLabel" name="nextPieceLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>10</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nextLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Muna PUA</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 71, 71);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameOverLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Noto Nastaliq Urdu UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(219, 33, 32);</string>
   </property>
   <property name="text">
    <string>GAME OVER!</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="aiMoveButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>128</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Muna PUA</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 207, 207);
border: none</string>
   </property>
   <property name="text">
    <string>Start AI Move</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heldPieceLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="heldLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Muna PUA</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 71, 71);</string>
   </property>
   <property name="text">
    <string>Held</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
